libnavigation-core/src/test/java/com/mapbox/navigation/directions/session/MapboxDirectionsSessionTest.kt

```diff
@@ -157,7 +158,7 @@ class MapboxDirectionsSessionTest {
         val throwable: Throwable = mockk()
         callback.onFailure(throwable)
         delayLambda()
-        verify(exactly = 1) { router.getRoute(any(), any()) }
+        verify(exactly = 2) { router.getRoute(any(), any()) }
```

Comment:
Why would we like to request a faster route if the original request failed? This should be handled by the user. Same as above, this is a change that I explicitly made and refs https://github.com/mapbox/mapbox-navigation-android/pull/2408/files#diff-2c6725b936f8c3d41f2d40c2ba2c2487L96.

Reply:
Now that I am thinking about it again, you are right. I will put this change back.